tests/metrics/test_metric.py

```diff
@@ -19,6 +22,13 @@
 class ColumnValuesAboveResult(MetricResult[bool]): ...
 
 
+class ColumnValuesAbove(ColumnMetric[ColumnValuesAboveResult]):
```

Comment:
Since we defined this in every test class I made this top level. In the definition tests, I renamed this to `MyColumnValuesAbove`.